sparse/tests/test_array_function.py

```diff
@@ -64,6 +64,7 @@ def test_ternary(func, arg_order):
     xx = func(x, x, x)
     args = [(x, y)[i] for i in arg_order]
     yy = func(*args)
+    sparse.save_npz("y.npz", y)
```

Comment:
Was this intended to be included in the PR?

Reply:
Whoops. No. I didn't know you kept an eye on this repo! I was debugging a particular test that failed in some scenarios so I dumped the array to repro it outside the test.

Reply:
Fixed in 62e35a5ac4677e2922c088aadbf27cf89c9050ae.